composer.json

```diff
@@ -16,7 +16,7 @@
   },
   "require": {
     "php": ">=7.4",
-    "monolog/monolog": "~1.25"
+    "monolog/monolog": "^2.0"
```

Comment:
As mentioned here (#1) the latest LTS version of Symfony does not support Monolog 2.
I will not merge a PR that would prevent using the 1.X version as long as the next LTS is not released.
Please adapt your PR to allow both version ~1.25 and ^2.0.
Then I can merge this change.